### DISCUSSION TITLE
v129

### DISCUSSION THREAD

**Discussion by dasgas12134** [3kh0/ext-remover#1187] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
Hey is there any exploits for extensions on v129? I have tried rigtools but they said it was patched I was just using it yesterday and it randomly freezes and restarted my Chromebook then I was on v129. It never said I needed to update my Chromebook.

**Comment by 88411** [comment]
Same happened to me -_-
Haven't found anything

**Comment by solanuscasey** [comment]
are you asking about adding or removing extensions?

**Reply by 88411** [reply]
Yes

**Reply by solanuscasey** [reply]
well aside from rigtools, what are some other tools that don't work anymore?

**Reply by 88411** [reply]
I mean basically everything. Iv'e gone through the whole list of exploits on this repo and tried them all and all of them are patched

**Reply by solanuscasey** [reply]
Interesting, I'll try to get around it, I assume I'm working under the same conditions as you(no crosh, terminal, or devmode on an administrated device) If I find something I'll let you know.

**Reply by dasgas12134** [reply]
> Interesting, I'll try to get around it, I assume I'm working under the same conditions as you(no crosh, terminal, or devmode on an administrated device) If I find something I'll let you know.

tysm

**Reply by 88411** [reply]
I do have access to crosh if that helps.

**Reply by solanuscasey** [reply]
https://developer.chrome.com/docs/extensions/how-to/distribute/install-extensions
take a look through there and see if any of this helps, I do not know if it is possible to run a JSON file through crosh, but it may be doable.

**Reply by 88411** [reply]
Thank you so much! I will look into this and let you know if I can get it to work.

**Reply by solanuscasey** [reply]
As far as I know, crosh will not be able to run this, but it may be possible to ssh(remote connect) to another home computer. if your school blocks websites that say crosh in the link, use the link preview flag to learn more.
